posthog/cdp/templates/avo/template_avo.py

```diff
@@ -0,0 +1,126 @@
+from posthog.cdp.templates.hog_function_template import HogFunctionTemplate
+
+
+template: HogFunctionTemplate = HogFunctionTemplate(
+    status="beta",
+    id="template-avo",
+    name="Send events to Avo",
+    description="Send events to Avo",
+    icon_url="/static/services/avo.png",
+    hog="""
+if (empty(inputs.apiKey) or empty(inputs.environment)) {
+    print('API Key and environment has to be set. Skipping...')
+    return
+}
+
+let avoEvent := {
+    'apiKey': inputs.apiKey,
+    'env': inputs.environment,
+    'appName': inputs.appName,
+    'sessionId': generateUUIDv4(),
```

Comment:
I see this is the same in the old plugin, but should we (eventually? theoretically?) use the `$session_id` property on the event?

Reply:
I've changed it to `event.properties.$session_id ?? generateUUIDv4()`